ws-calculator/src/main/resources/application.properties

```diff
@@ -31,7 +31,7 @@ ws.calculator.demand.successful.topic=ws-demand-saved
 ws.calculator.demand.failed=ws-demand-failure
 
 ##meterReading Default Limit And Offset
-egov.ws_calculation.meterReading.default.limit=10
+egov.ws_calculation.meterReading.default.limit=50
```

Comment:
What is this limit on meterReading all about? This name looks wrong, if you are using it for some list AP, name this property properly. Currently it looks like a limit on what meter reading some one can enter